### ISSUE TITLE
Convert all projects from spaces to tabs

### ISSUE THREAD

**Issue by Gummibeer** [Astrotomic/.github#10]
As of preference and accessibility the width of a tab/indentation should be adjustable without altering the code itself.
That's why all projects should be converted to tabs and CS rules adjusted properly to enforce tabs forever.

**Comment by himanshu007-creator** [comment]
Hi, i would like to work on this issue
